README.md

```diff
@@ -4,4 +4,58 @@ cnab-go is a library for building [CNAB](https://github.com/deislabs/cnab-spec)
 
 cnab-go is currently being used by [Docker App](https://github.com/docker/app), [Duffle](https://github.com/deislabs/duffle), and [Porter](https://github.com/deislabs/porter). If you'd like to see your CNAB project listed here, please submit a PR.
 
+### Community
+
 cnab-go is [maintained](GOVERNANCE.md) by the CNAB community. We sometimes discuss cnab-go issues during the [bi-weekly CNAB community  meeting](https://hackmd.io/s/SyGcBcwQ4), but we encourage open communication via our [issue](https://github.com/deislabs/cnab-go/issues) queue and via [PRs](https://github.com/deislabs/cnab-go/pulls). If you are interested in contributing to cnab-go, please refer to our [contributing](CONTRIBUTING.md) guidelines.
+
+### Development
+
+#### Getting the code
+
+Cloning this repository and change directory to it:
+```bash
+$ go get -d github.com/deislabs/cnab-go/...
+$ cd $(go env GOPATH)/src/github.com/deislabs/cnab-go
```

Comment:
This will change as soon as #52 is merged. Perhaps we should get that in first?

Reply:
Maybe not be so specific about where to clone it?

Reply:
I think I agree with @jlegrone that maybe this change here should wait for #52 to be merged before merging this one.
And when we that happens we can remove some of these lines 😄